cirq-web/cirq_ts/src/bloch_sphere/components/Meridians.class.ts

```diff
@@ -0,0 +1,146 @@
+import {EllipseCurve, BufferGeometry, LineBasicMaterial, Line} from 'three';
+
+export class Meridians {
+  private static _curveData = {
```

Comment:
The purpose of the fns here is to output an array of `Line` objects that can easily be added to the three.js bloch sphere `Group`..are you suggesting that I wrap the output in a `Meridian` type with the `Line` info inside? IMO it's clear just to output the line objects and add..on the other end we'd have to unpack them. It's not that big of a deal but just seems like an extra step. I can see tho how it would be clearer for organization.

What do you think?